services/terrain/src/terrain/clients/apps/raw.clj

```diff
@@ -582,6 +582,27 @@
                :as               :stream
                :follow-redirects false}))
 
+(defn get-oauth-token-info
+  [api-name]
+  (client/get (apps-url "oauth" "token-info" api-name)
+              {:query-params     (secured-params)
+               :as               :stream
+               :follow-redirects false}))
+
+(defn get-oauth-redirect-uris
+  []
+  (client/get (apps-url "oauth" "redirect-uris")
+              {:query-params     (secured-params)
+               :as               :stream
```

Comment:
Since this is always being decoded as JSON in terrain, and the response isn't forwarded to the client, we could change this to `:as :json` here and remove the `service/decode-json` call on these results.